src/makeRobot.js

```diff
@@ -38,7 +38,64 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
+    'coords': {
+      'x': 0,
+      'y': 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, `
+      + `chip version: ${this.version}, `
+      + `wheels: ${this.wheels}`;
+    },
```

Comment:
maybe better to have one solid string without concatenation. For example like this:

```suggestion
        get info() {
      return `name: ${
        this.name
      }, chip version: ${
        this.version
      }, wheels: ${
        this.wheels
      }`;
    },
```

Reply:
but it is only a suggestion. Mentors can correct you or me.

Reply:
Both variants have the right to live. But the first variant is easier to read